Makefile

```diff
@@ -90,39 +78,15 @@ release: ## Builds cross-compiled binaries
 		-o $(BUILDDIR)/$$dir -a $(PKG)/$(FUNCDIR)/$$dir; \
 	done
 
-.PHONY: package
-package: release ## Packages and uploads CloudFormation stack ressources to S3
-	@echo "+ $@"
-	@aws cloudformation package \
-	--template-file template.yaml \
-	--s3-bucket $(BUCKET) \
-	--s3-prefix $(NAME) \
-	--output-template-file $(BUILDDIR)/packaged.yaml \
-	--region $(REGION)
-
 .PHONY: deploy
-deploy: clean package ## Creates/updates CloudFormation stack with termination protection
+deploy: ## Builds or changes infrastructure using Terraform
 	@echo "+ $@"
-	@aws cloudformation deploy \
-	--template-file $(BUILDDIR)/packaged.yaml \
-	--stack-name $(NAME) \
-	--capabilities CAPABILITY_IAM \
-	--region $(REGION)
-	@aws cloudformation update-termination-protection \
-	--enable-termination-protection \
-	--stack-name $(NAME) \
-	--region $(REGION)
-
-.PHONY: delete-stack
-delete-stack: ## Disabled termination protection and deletes CloudFormation stack
+	@ cd terraform && terraform apply
```

Comment:
`terraform plan`?

Reply:
Ist schwierig mit solchen modulen. Das Problem wäre mir hier zu klein, um ein TF-Modul dafür zu verwenden. Und gerade dort, wo man vielleicht sehr individuel anpassen muss, kommt man schnell an Grenzen.

* `env` variables in Verbindung mit KMS Verschlüßelung
* IAM/Rollen greifen mir etwas zu kurz hier.

Wie schon in dem einen Kommentar gesagt, die wenigen TF Dateien können direkt ins Projekt, wie z.B. bei https://github.com/spring-media/rbbt-abe/blob/master/terraform/